src/fastoad/notebooks/03_custom_modules/beam_problem.ipynb

```diff
@@ -0,0 +1,271 @@
+    "  <div class=\"column\">\n",
+    "    <img src=\"./img/problem_description.png\" width=\"600\">\n",
+    "  </div>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os.path as pth\n",
+    "import logging\n",
+    "import fastoad.api as oad"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Directories and files definition "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DATA_FOLDER = \"data\"\n",
+    "\n",
+    "WORK_FOLDER = \"workdir\"\n",
+    "\n",
+    "CONFIGURATION_FILE_NAME = pth.join(DATA_FOLDER, \"beam_problem.yml\")\n",
+    "\n",
+    "logging.basicConfig(level=logging.INFO, format='%(levelname)-8s: %(message)s')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Custom Modules\n",
+    "\n",
+    "The following modules are created: [section_properties.py](./modules/section_properties.py), [weight.py](./modules/weight.py), [displacements.py](./modules/displacements.py), [stresses.py](./modules/stresses.py). They respectively compute the heigth of the beam for a given moment of inertia and width, the linear weight **w** from material density and section dimensions, the necessary moment of inertia to reach a given displacements and the necessary moment of inertia not to overcome yield stress. \n",
+    "\n",
+    "As you may notice, each module is registered using the decorator `@oad.RegisterOpenMDAOSystem`. \n",
+    "\n",
+    "An example is provided for a geometric module that compute the height of the beam for a given moment of inertia and a given width: \n",
+    "\n",
+    "<div class=\"row\">\n",
+    "    <div class=\"column\">\n",
+    "        <img src=\"./img/custom_module_ex.gif\" width=\"600\">\n",
+    "  </div>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Configuration file\n",
+    "Once the modules are created, the problem can be defined through the [configuration file](./data/beam_problem.yml): \n",
+    "* First, the folder containing your custom module must be indicated within the `module_folders` section.\n",
+    "* Then, your problem is defined within the `model` section. \n",
+    "* If variables are shared between different modules (models) then loops occurs and `nonlinear_solver` and `linear_solver` must be defined, here non linear Block Gauss-Seidel and linear Direct solvers are used. \n",
+    "The configuration file for the first problem is shown below. As you can notice, three different modules are called sequentially: the geometry, the weight and the displacements. As Displacements and geometry shares inputs/outputs we need to introduce solvers. "
```

Comment:
1. I would suggest "As you can notice, three modules are called sequentially: `geometry`, `weight` and `displacements`. As `displacements` and `geometry` ...". 
2. The "s" of "shares" is incorrect.
3. This "sharing variables" formulation should be rewritten, as mentioned above.